src/main.cpp

```diff
@@ -4344,8 +4344,8 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         int nDoS = 0;
         if (state.IsInvalid(nDoS))
         {
-            LogPrint("mempoolrej", "%s from peer=%d %s was not accepted into the memory pool: %s\n", tx.GetHash().ToString(),
-                pfrom->id, pfrom->cleanSubVer,
+            LogPrint("mempoolrej", "%s from peer=%d was not accepted into the memory pool: %s\n", tx.GetHash().ToString(),
```

Comment:
Now that we're changing this message: can we shorten it even further? I introduced this, but as it appears quite often (when mempoolrej enabled), I'd prefer a shorter formulation. Maybe "was not accepted" is enough?